src/components/map/PopupContent.svelte

```diff
@@ -1,10 +1,11 @@
 <script>
-	let { stopName, arrivalTime } = $props();
+	import { t } from 'svelte-i18n';
+	let { stopName, arrivalTime, handleStopMarkerSelect } = $props();
 </script>
 
 <div class="transform rounded-lg bg-white p-4 shadow-lg">
 	<h3 class="text-xl font-bold text-gray-900">{stopName}</h3>
-	<p class="mt-2 flex items-center text-gray-700">
+	<p class="my-4 flex items-center text-gray-700">
```

Comment:
I think this popup could use a styling pass, but this at least made the Google version a bit less scrunched together.